### ISSUE TITLE
Add container group data model

### ISSUE THREAD

**Issue by reubenmiller** [thin-edge/tedge-container-plugin#7]
**Example service data model**

The following show an example of addition container-group information that can be added to the service.

```jsonc
{
  "name": "example::redis", // added via service registration in format of "{projectName}::{serviceName}"
  "containerId": "1f3f0c98877a",
  "containerName": "example_redis_1",
  "projectName": "example",
  "serviceName": "redis",
  "ports": "8080",
  "command": "\"/bin/sh -c 'echo Container started\\n trap \\\"exit 0\\\" 15\\n \\n exec \\\"$@\\\"\\n while sleep 1 & wait $!; do :; done' - /lib/systemd/systemd --system --unit=basic.target\"",
  "networks": "thin-edgeio_devcontainer_default",
  "filesystem": "2.73GB (virtual 5.66GB)",
  "image": "thin-edgeio_devcontainer-tedge",
  "runningFor": "3 weeks ago",
  "state": "running",
  "status": "Up 13 days",
}
```

**Comment by reubenmiller** [comment]
Resolved in 0.3.0

**Comment by reubenmiller** [comment]
The data schema has also been added to the [TELEMETRY](https://github.com/thin-edge/tedge-container-plugin/blob/main/docs/TELEMETRY.md#container-groups-container-group) docs.
